docs/bump-upstream.md

```diff
@@ -8,13 +8,15 @@
    - [`src/Splunk.OpenTelemetry.AutoInstrumentation/Splunk.OpenTelemetry.AutoInstrumentation.csproj`](../src/Splunk.OpenTelemetry.AutoInstrumentation/Splunk.OpenTelemetry.AutoInstrumentation.csproj)
    - [`test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/SmokeTests.cs`](../test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/SmokeTests.cs)
 
-2. Update the `test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/BuildTests.DistributionStructure_*.verified.txt`
+1. Update the `test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/BuildTests.DistributionStructure_*.verified.txt`
    files.
 
-3. Update the [required env vars table](./advanced-config.md#manual-instrumentation).
+1. Update the [required env vars table](./advanced-config.md#manual-instrumentation).
 
-4. Update the scripts based on changes in upstream:
+1. Update the scripts based on changes in upstream:
    - [`splunk-otel-dotnet-install.sh`](../splunk-otel-dotnet-install.sh)
    - [`Splunk.OTel.DotNet.psm1`](../Splunk.OTel.DotNet.psm1)
 
-5. Update the [GitHub workflows](../.github/workflows) on changes in upstream.
+1. Update the [GitHub workflows](../.github/workflows) on changes in upstream.
+
+1. Update internal repository. Follow internal documentation.
```

Comment:
This should go to https://github.com/signalfx/splunk-otel-dotnet/blob/main/RELEASING.md